ArduPlane/altitude.cpp

```diff
@@ -358,26 +358,37 @@ int32_t Plane::calc_altitude_error_cm(void)
 }
 
 /*
-  check for FBWB_min_altitude_cm violation
- */
-void Plane::check_fbwb_minimum_altitude(void)
+  check for min and max altitude: FBWB_min_altitude_cm and fbwb_max_altitude_m violation
+*/
+void Plane::check_fbwb_altitude(void)
 {
-    if (g.FBWB_min_altitude_cm == 0) {
+    float max_alt_cm = fence.get_safe_alt_max()*100.0f;
```

Comment:
need to check that the max alt fence is enabled, so that these numbers are valid, default is polygon only

Reply:
@tridge sir, I know, but the immediate next instruction does that (checking is it is enabled). Also, the minimum altitude is considered by the param `ALT_HOLD_FBWCM` rather than fence min altitude. Maximum from the fence, max altitude.
Is that you are concerned about in this feedback ?